locales/en/manage-kafka-permissions.json

```diff
@@ -48,7 +43,25 @@
     "task_based_permission": "Task-based permissions"
   },
   "remove_permission_tooltip": "Remove permission",
+  "resource_prefix_rule": {
+    "aria_label": "Resource prefix rule {{value}}",
+    "is": "Is",
+    "is_description": "Refers to the literal name in the API",
+    "starts_with": "Starts with",
+    "starts_with_description": "Refers to the prefix of the name in the API"
+  },
   "resource_type_operation_filter_placeholder": "Filter by resource type operation",
+  "resourcePrefix": {
+    "create_text": "Use",
+    "invalid_consumer_group": "Valid characters in a consumer group ID include letters (Aa–Zz), numbers, underscores ( _ ), and hyphens ( - ).",
+    "invalid_length": "Cannot exceed 32 characters",
+    "invalid_topic": "Valid characters in a topic name include letters (Aa-Zz), numbers, underscores ( _ ), periods ( . ), and hyphens ( - ).",
+    "invalid_topic_length": "A topic name must contain at least 3 periods (...) if periods are the only characters used",
```

Comment:
One other minor comment—
The string `invalid_length` is used when the user enters too many characters, but this message is not about length.
This message displays if the user specifies either "." or ".." when "Is" is selected. Maybe `invalid_topic_name` is more applicable here?